x/community/genesis.go

```diff
@@ -11,9 +11,11 @@ import (
 )
 
 // InitGenesis initializes the community module account
-func InitGenesis(ctx sdk.Context, k keeper.Keeper, ak types.AccountKeeper) {
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, ak types.AccountKeeper, gs *types.GenesisState) {
 	// check if the module account exists
 	if moduleAcc := ak.GetModuleAccount(ctx, types.ModuleAccountName); moduleAcc == nil {
 		panic(fmt.Sprintf("%s module account has not been set", types.ModuleAccountName))
 	}
+
+	k.SetParams(ctx, gs.Params)
```

Comment:
assuming this will need a migration for public testnet and mainnet, @nddeluca is this an example of a "consensus" level migration that should be written and tested in the current PR?

Reply:
yes, we will need a module level migration to set the default params and we can add some tests for this. I'll add it to the PR.